charts/hivemq-platform/tests/services/hivemq_control_center_services_test.yaml

```diff
@@ -145,3 +145,26 @@ tests:
       - notFailedTemplate: {}
       - hasDocuments:
           count: 1
+
+  - it: with a Control Center service enabled and HiveMQ listener name enabled, validation fails
+    set:
+      services:
+        - type: control-center
+          exposed: true
+          containerPort: 8081
+          hivemqListenerName: foobar
+    asserts:
+      - failedTemplate:
+          errorPattern: Service type `control-center` with container port `8081` is using `hivemqListenerName` value. HiveMQ listener names are only supported by MQTT and WebSocket services
```

Comment:
Same here, why is the container port necessary here.

Reply:
Same as explained already.